src/components/Comments.jsx

```diff
@@ -1,7 +1,41 @@
-import React from "react";
+import React, { useState } from "react";
+import { v4 as uuidv4 } from "uuid";
+import CommentCard from "./CommentCard";
+import Toggle from "./Toggle";
+import "../styles/Comments.css";
 
-function Comments({ articleId, setArticleId }) {
-  return <main>Comments</main>;
+function Comments({ comments }) {
+  const [isHidden, setIsHidden] = useState(true);
```

Comment:
I would fetch comments in this component rather than in article and have them in state here